bpf/bpf_alignchecker.c

```diff
@@ -27,6 +25,7 @@
 #include "lib/conntrack.h"
 #include "lib/dbg.h"
 #include "lib/drop.h"
+#define SKIP_UNDEF_LPM_LOOKUP_FN
```

Comment:
Was this related, or just another cleanup that got caught up in this PR?

Reply:
Just another cleanup which was surfaced by several functions suddenly being complained about by the compiler as being unused after this change. I'm still not sure why there weren't any `-Wunused-function` errors before this change though.